src/InterventionBackend.php

```diff
@@ -810,10 +672,6 @@ protected function hasFailed($hash, $variant = null)
      */
     public function __destruct()
     {
-        //skip the `getImageResource` method because we don't want to load the resource just to destroy it
-        if ($this->image) {
-            $this->image->destroy();
-        }
```

Comment:
https://github.com/Intervention/image/issues/1273 That method is no longer available and there's no replacement.